HipercowApi/HipercowApi.csproj

```diff
@@ -5,12 +5,18 @@
     <Nullable>enable</Nullable>
     <ImplicitUsings>enable</ImplicitUsings>
     <RootNamespace>Hipercow_api</RootNamespace>
-	<EnableUnsafeBinaryFormatterSerialization>true</EnableUnsafeBinaryFormatterSerialization>
-	<GeneratePackageOnBuild>False</GeneratePackageOnBuild>
-	<Copyright>Imperial College London</Copyright>
+    <EnableUnsafeBinaryFormatterSerialization>true</EnableUnsafeBinaryFormatterSerialization>
+    <GeneratePackageOnBuild>False</GeneratePackageOnBuild>
+    <Copyright>Imperial College London</Copyright>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
   </PropertyGroup>
 
+  <ItemGroup>
+    <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
+      <_Parameter1>HipercowApiUnitTests</_Parameter1>
+    </AssemblyAttribute>
+  </ItemGroup>
+
```

Comment:
This was because I wanted to test some private functions, but couldn't because they were... private! I changed them to `internal` - and apparently this then exposes them to the test project so I can call them... is that a reasonable way?

Reply:
yup perfect